File: src/main/resources/export/java/PIDCommand-output.java
#set($command = $helper.getByName($command_name, $robot))
#set($component = $command.getProperty("Output").getValue())
#set($name = ${helper.getByName($component, $robot).fullName})
#set($subsystem = $command.getProperty("Requires").getValue())
    #if($name)        return Robot.#variable($subsystem).get#class($name)().pidWrite(output);
#end

File: src/main/resources/export/cpp/PIDCommand-output.cpp
#set($command = $helper.getByName($command_name, $robot))
#set($actuator = $command.getProperty("Output").getValue())
#set($subsystem = $command.getProperty("Requires").getValue())
#foreach ($component in $components)
#if ($component.name == $actuator)
    Robot::#variable($subsystem)->Get#class($component.name)()->PIDWrite(output);
#end
#end